website/docs/terms/reverse-etl.md

```diff
@@ -83,6 +83,6 @@ Reverse ETL enables you to sync your transformed data stored in your data wareho
 
 If you’re interested learning more about reverse ETL and the impact it could have on your team, check out the following:
 
-- [How dbt Labs’s Data Team Approaches Reverse ETL](https://getdbt.com/open-source-data-culture/reverse-etl-playbook/)
-- [The Operational Data Warehouse in Action: Reverse ETL, CDPs, and the Future of Data Activation](https://www.getdbt.com/coalesce-2021/operational-data-warehouse-reverse-etl-cdp-data-activation/)
-- [The Analytics Engineering Guide: Operational Analytics](https://www.getdbt.com/analytics-engineering/use-cases/operational-analytics/)
+- [How dbt Labs’s data team approaches reverse ETL](https://getdbt.com/open-source-data-culture/reverse-etl-playbook/)
```

Comment:
This is non-blocking, but maybe we can follow up next week with a short sentence identifying why a reader would benefit form reading the linked resource?

Reply:
Yeah we're actually working on creating more consistent CTAs/next steps from glossary pages and other content soon, so we should sync!

Reply:
Thx for the review Leona! sorry about the weird package json stuff 🤷